iCub_manual/conf_manual/iCubGazeboV3_visuomanip/gazebo_icub_left_hand.ini

```diff
@@ -0,0 +1,23 @@
+[include "gazebo_icub_robotname.ini"]
+
+[WRAPPER]
+# name of the wrapper device to be instatiated by the factory
+device controlboardwrapper2
+# rate of output streaming from ports in ms
+period 10
+# output port name (a leading '/' will be added by code)
+name /${gazeboYarpPluginsRobotName}/left_hand
+# Total number of joints
+joints 9
+
+# list of MotorControl device to use
+networks ( left_hand_thumb left_hand_index left_hand_middle left_hand_pinky left_hand_finger)
+# for each network specify the joint map
+left_hand_thumb 0 2 0 2
+left_hand_index 3 4 0 1
+left_hand_middle 5 6 0 1
+left_hand_pinky 7 7 0 0
+left_hand_finger 8 8 0 0
+
+# Verbose output (on if present, off if commented out)
+#verbose
```

Comment:
If I am not wrong, this file is unused. Can we delete it?

Reply:
indeed you are correct. Left-over files from copy/paste.  Fixed with 494367d2c695608b9b140170a20d225843dec4f3